README.md

```diff
@@ -76,29 +76,16 @@ restart as well.
 
 You can pull our Docker images from [Docker Hub](https://hub.docker.com/r/timescale/timescaledb/).
 
-```bash
-docker pull timescale/timescaledb:latest
-```
+To run, simply issue the following command:
 
-To run, you'll need to specify a directory where data should be
-stored/mounted from on the host machine. For example, if you want
-to store the data in `/your/data/dir` on the host machine:
 ```bash
-docker run -d \
-  --name timescaledb \
-  -v /your/data/dir:/var/lib/postgresql/data \
-  -p 5432:5432 \
```

Comment:
So, this was a long discussion between me, Rob, and Ajay when we originally wrote these instructions. We decided not to provide a command without a volume mount to prevent unexpected data loss for people. Users might not be that familiar with docker or how volumes work, and including volume mounts seemed like the "safer way to go".